src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -32,15 +36,20 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
     public readonly contextValue: string = PostgresServerTreeItem.contextValue;
     public readonly childTypeLabel: string = "Database";
     public readonly server: Server;
+    public readonly connectionString: ParsedPostgresConnectionString;
     public resourceGroup: string;
 
     private _serverId: string;
 
-    constructor(parent: AzureParentTreeItem, server: Server) {
+    constructor(parent: AzureParentTreeItem, server?: Server, connectionString?: ParsedPostgresConnectionString) {
```

Comment:
Similar to another comment, but the connectionString should be the preferred way of doing things because it works for attached and non attached. So you should make `connectionString` a required parameter and move it before `server`. Also you should use `server` as little as possible. Similar to what MongoAccountTreeItem does

Reply:
For CosmosDB accounts, CosmosDBManagmentClient provides `listConnectionsStrings` for Mongo accounts. However, for Postgres accounts we don't have access to postgres connection strings through the PostgreSQLManagmentClient, we only have access to Server or Database. However, we can create the connection string and add it to the connectionString field but it won't be initialized in the constructor but in the loadMore... function of ...ServerTreeItem. Would that be ok?